Microverse Portfolio

masangana opened the pull request:
This PR is raised to add the main page to the portfolio
### Description of the task to be completed
- Used Github workflow
- Configured the project to use linters
- Created the main page
- styled the main page
- Added a descriptive README
- Fixed linter errors
- Added a descriptive PR

masangana committed:
add html and css

-- creation nav bar and collapsed-menu
-- basic menu style for test

masangana committed:
fix css and html

adapting the project to the template without use linters.

--adding css in the nav bar and main section

masangana committed:
fix css and html

--adding css codes to approach the template
-- Fixing elements color
-- Fixing elements size
-- Fixing Html incoherence

masangana committed:
fix linters css errors

fixing css errors

phelian23 commented on index.html:
```diff
@@ -0,0 +1,70 @@
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <div class="logo">
+            <a href="">Alexander</a>
+        </div>
+        <ul class="top-navigation">
+            <li class="navigation-item">
+                <a href="">Portfolio</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">About</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">Contact</a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
+    </nav>
```
- [ ] For semantic purposes, kindly wrap the toolbar section in a `<header>` tag instead of the `<nav>` tag. This will help improve the SEO of your page

phelian23 commented on index.html:
```diff
@@ -0,0 +1,70 @@
+        <div class="logo">
+            <a href="">Alexander</a>
+        </div>
+        <ul class="top-navigation">
+            <li class="navigation-item">
+                <a href="">Portfolio</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">About</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">Contact</a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
```
- [ ] Kindly wrap the nav items in a `<nav>` tag.

masangana replied:
checked 👍

phelian23 commented on index.html:
```diff
@@ -0,0 +1,70 @@
+
+    <main class="main" >
+        <div class="headline">
+            <div class="social-media">
+                <a href=""><img src="./ressources/icons/twitter.svg" alt="logo_twitter" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/github.svg" alt="logo_github" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/Linkedin.svg" alt="logo_Linkedin" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/angel.svg" alt="logo_angel" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/medium.svg" alt="logo_medium" 
+                    class="logo-social">
+                </a>
+            </div>
```
- [ ] For semantic purposes try to wrap the list of icons in a `<ul>` tag and each one should be `<li>` tag as improve your SEO benchmark and your section will be structured
example :
``` markup
<ul>
  <li>facebook icon</li>
  <li>twitter icon</li>
</ul>
```

masangana replied:
chanded 👍

phelian23 commented on index.html:
```diff
@@ -0,0 +1,70 @@
+        </div>
+    </nav>
+
+    <main class="main" >
+        <div class="headline">
+            <div class="social-media">
+                <a href=""><img src="./ressources/icons/twitter.svg" alt="logo_twitter" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/github.svg" alt="logo_github" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/Linkedin.svg" alt="logo_Linkedin" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/angel.svg" alt="logo_angel" 
+                    class="logo-social">
```
- [ ] Kindly wrap the headline section in a `<section>` tag.

masangana replied:
done 👍

phelian23 commented on index.html:
```diff
@@ -0,0 +1,70 @@
+                </a>
+            </div>
+            <div class="text-headline">
+                <p class="primery-text">
+                    Hey There. <br>I'm Alex
+                </p>
+                <p class="supporting-text">
+                    I’am a Software Developer
+                </p>
+                <p class="description">
+                    I can help you build a product , 
+                    feature or website Look through some of my work and experience! I
+                    f you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            
```
- [ ] Kindly make sure that your design (style, margins, fonts family, background images) matches that of the figma template as much as possible.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/164451172-fb083169-b404-4856-a66a-083387832cbe.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/164451271-ba1c6353-546f-4a05-a425-ac26bb9edb64.PNG)

masangana replied:
done 👍 i have just forgeted this

phelian23 reviewed (changes requested):
Hi @masangana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

masangana committed:
fix html and css issue

fixing
--adding html balise (nav, head, main)
--adding (backgroud image)
--completing style requirement
--rebuilding css structure for new html organization

masangana committed:
fix linters errors

fixing
--css errors
--html errors

masangana committed:
fix linters html errors

--remove some empty elements

masangana committed:
fix html issues

--  remove the empty value for id

masangana commented:
### @phelian23 
Hello. Can you please check the corrections before i submit the new review request.
Waiting for your feedback

codepantha commented on index.html:
```diff
@@ -0,0 +1,100 @@
+        <main id="main">
+            <!--home page-->
+            <section class="text-headline">
+              <h1 class="primery-text">Hey There.<br />I'm Alexander</h1>
+      
+              <div class="main-description">
+                <h2 class="supporting-text">I'm a Software Developer</h2>
+      
+                <p class="description">
+                  I can help you build a product, feature or website Look through some
+                  of my work and experiece! If you like what you see and have a
+                  project you need coded, don't hesitate to contact me.
+                </p>
+              </div>
```
- [ ] As the previous reviewer requested, please try to adjust your background image to conform to the figma design 🙏 
### Your design
![image](https://user-images.githubusercontent.com/46839548/164548121-448822c3-f8f9-4cbc-b311-fd606e6a147c.png)
### Figma Design
![image](https://user-images.githubusercontent.com/85095702/164451271-ba1c6353-546f-4a05-a425-ac26bb9edb64.PNG)

masangana replied:
### @codepantha 
hello [screen shoot](https://drive.google.com/file/d/1R1b3x3CzzRYjBuNd8KTpHPRhdjAOA6wr/view?usp=sharing).
Due to the screen size the position can change. you can see here de result that i have from my computer.
waitting for your feed back

### @phelian23 
Please can you take a look

codepantha replied:
Alright @phelian23. I still think the placement is not the best in smaller devices like `320px` and `375px` but we can work with this. Please go ahead and request for another review.

masangana replied:
i have made my last review request

codepantha reviewed (changes requested):
Hi @masangana 👋, 

## STATUS: CHANGES REQUESTED ♻️ 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://media3.giphy.com/media/Qaxd5XpgDWMso/giphy.gif"/>

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

masangana committed:
fix css background

setting the background to the bottom right position

masangana committed:
fix css background

setting the background in the bottom right

masangana committed:
fix css

checking main container

masangana committed:
fix css

remove useless lines

mahbubul14 reviewed (approved):
# Project Approved :trophy: :tada: 

Hi @masangana , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Well implemented Figma design.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
